pyro/distributions/transforms/simplex_to_ordered.py

```diff
@@ -0,0 +1,80 @@
+# Copyright Contributors to the Pyro project.
+# SPDX-License-Identifier: Apache-2.0
+
+import torch
+from torch.distributions.transforms import Transform
+
+from .. import constraints
+
+
+def expit(x):
+    return 1.0 / (1 + torch.exp(-x))
+
+
+def logit(x):
+    return torch.log(x / (1.0 - x))
```

Comment:
Could you replace these with `torch.special.expit()` and `torch.specical.logit()` which should be more numerically stable?

Reply:
Of course! I hadn't realized these were available.